### ISSUE TITLE
Model training jobs stuck/error

### ISSUE THREAD

**Issue by koonchaya** [rfcx/arbimon#2139]
From: helmigkylie@gmail.com

Project: https://arbimon.org/p/cicadascapes

>I put a test set of data into Arbimon and was just playing around with the different modes of analysis that this program can do, and am currently still waiting on the two model training sets I initiated. Whenever they first began running they both jumped to 89.8% completed, and now after 2 weeks, they are sitting in the same 89.8% completed status.

![Image](https://github.com/user-attachments/assets/d56f0535-594d-44e2-8fc3-ec4603c2bfbf)
